crates/matrix-sdk/src/room/timeline/virtual_item.rs

```diff
@@ -0,0 +1,8 @@
+/// A [`TimelineItem`](super::TimelineItem) that doesn't correspond to an event.
+#[derive(Clone, Debug)]
+pub enum VirtualTimelineItem {
+    /// A divider between messages of two days.
+    DayDivider,
```

Comment:
It would be nice if this included the date of which day it represents (again maybe coming, but just thinking out loud here).